### PR TITLE
Add Azure goat

### PR DESCRIPTION
**Pull request by nimrodkor**
This is the first phase - adding just a few resources and moving the AWS files. Will go back and update the README after I finish Azure

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ see: [checkov](https://github.com/bridgecrewio/checkov/), the open source static
 
 ## Getting started
 ### Installation
-You can deploy multiple TerraGoat stacks in a single AWS account using the parameters `TF_VAR_environment` and `TF_VAR_environment`.
+You can deploy multiple TerraGoat stacks in a single AWS account using the parameter `TF_VAR_environment`.
```
Comment (schosterbarak) [inline README.md:40]: can you update how to deploy azure too?

Comment (schosterbarak) [reply README.md:40]: and tag a new release version please :)

**Comment by schosterbarak** [comment]
isn't terraform/resources/lambda_function_payload.zip required for lambda provisioning?

**Comment by nimrodkor** [comment]
@schosterbarak It is, and it can now be found in `terraform/aws/resources/lambda_function_payload.zip`

File: README.md (hunk 1 above)
Comment (nimrodkor) [reply README.md:43]: Once I release, sure

**Pull request merged**